README.md

```diff
@@ -303,6 +303,56 @@ users, so make sure to understand what you are doing.
 * `doil mail:stop` stops the mail server
 * `doil mail:restart` restarts the mail server
 
+### xdedug
+
+**doil** provides two options to enable xdebug for the given instance.  
+
+First an instance can be created with the '-x' flag, or alternatively 
+in interactive mode you will be asked to install xdebug.  
+On the command line it could look like this:  
+* `doil create -n -e ilias7 -r ilias -b release_7 -p 7.4 -u -x`
+
+Second, you can apply a state to an already existing instance.  
+To activate xdebug use the following command:
+
+* `doil apply <instance_name> enable-xdebug`
+
+Alternatively, you can type `doil apply <instance_name>` and select
+'enable-xdebug' from the list.
+
+You can turn off xdebug again in the same way:
+
+* `doil apply <instance_name> disable-xdebug`
+
+xdebug listens to port 9000 when activated. Now the client has to be set up. 
+Unfortunately, I can only present the configuration of PHP-Storm here as an example.
+But it should work similarly for other editors.
+
+#### Necessary PHP-Storm plugins:
+
+* PHP Docker
+* PHP Remote
+
+#### PHP-Storm settings
+
+* it is best to open the doil instance root directory in PHP-Storm.
+* under Settings PHP CLI interpreter: add a new interpreter (on the 3 dots and then in the pop up on +)
+* select "From Docker, Vagrant..." and then select "docker-compose".
+* if you already have the doil instance root open in phpstorm, it automatically selects the docker-compose.yml
```

Comment:
We should call it "PhpStorm" everywhere to be consistent

Reply:
Done!